TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/datahandlers/http/RequestPipelineOrchestrator.java

```diff
@@ -77,30 +78,41 @@ void addContentParsingHandlers(ChannelPipeline pipeline, JsonTransformer transfo
             log.warn("Adding JSON handlers to pipeline");
             //  IN: Netty HttpRequest(2) + HttpJsonMessage(1) with headers + HttpContent(2) blocks
             // OUT: Netty HttpRequest(2) + HttpJsonMessage(2) with headers AND payload
+            addLoggingHandler(pipeline, "B");
             pipeline.addLast(new NettyJsonBodyAccumulateHandler());
             //  IN: Netty HttpRequest(2) + HttpJsonMessage(2) with headers AND payload
```

Comment:
It isn't today, out of the box, because for C & D, the message flowing through is just an HttpJsonMessage, which is going to look the same at each stage because it isn't dumping the body.  We could make some further edits to elaborate.  That would be useful, but a bit more involved.